packages/core/src/helpers/calls.ts

```diff
@@ -20,14 +20,41 @@ export function warnOnInvalidCall(call: Call | Falsy) {
 /**
  * @internal Intended for internal use - use it on your own risk
  */
-export function encodeCallData(call: Call | Falsy, chainId: number, queryParams: QueryParams = {}): RawCall | Falsy {
+export interface InvalidRawCall {
+  chainId: ChainId
+  address: string
+  data: string
+  isValid: boolean
```

Comment:
This field does not make sense - there is no `InvalidRawCall` with `isValid=true`

Reply:
How about - `InvalidRawCall extends RawCall`
with an additional field `errorString`